cli/network.go

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2018 Huawei Corporation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/kata-containers/agent/protocols/grpc"
+	vc "github.com/kata-containers/runtime/virtcontainers"
+	"github.com/urfave/cli"
+)
+
+var networkCLICommand = cli.Command{
+	Name:  "network",
```

Comment:
I don't see a `network` command in `runc`, I think this is `kata` specific and it should be named `kata-network`

Reply:
Yes. Fixed.